### DISCUSSION TITLE
Multi Message List SysEx

### DISCUSSION THREAD

**Discussion by Benis67** [RomanKubiak/ctrlr#300] [Q&A]
I'm working on a simple Editor for roland Zen-Core synths (Fantom, JupiterX, AX Edge, ..).
To facilitate rapid programming, I am trying (optionally) to send the same value to 4 different parameters via Slider - SysEX - Multi Message list. Example fine tune a partial 1,2,3 and 4
Unfortunately only the first message gives correct values; the following always supply the variable = 0
Multi Message list:
SysEx,Direct,Direct,-1,-1,f0 41 10 00 00 00 52 12 01 01 20 02 xx 00 f7
SysEx,Direct,Direct,-1,-1,f0 41 10 00 00 00 52 12 01 01 21 02 xx 00 f7
SysEx,Direct,Direct,-1,-1,f0 41 10 00 00 00 52 12 01 01 22 02 xx 00 f7
SysEx,Direct,Direct,-1,-1,f0 41 10 00 00 00 52 12 01 01 23 02 xx 00 f7

Output
[16:10:27:000238]: No:[----] Val:[----] RAW:[f0 41 10 00 00 00 52 12 01 01 20 02 11 00 f7]  11 is the correct value
[16:10:27:000238]: No:[----] Val:[----] RAW:[f0 41 10 00 00 00 52 12 01 01 21 02 00 00 f7]  00 instead of 11
[16:10:27:000238]: No:[----] Val:[----] RAW:[f0 41 10 00 00 00 52 12 01 01 23 02 00 00 f7]  00 instead of 11
[16:10:27:000238]: No:[----] Val:[----] RAW:[f0 41 10 00 00 00 52 12 01 01 23 02 00 00 f7]  00 instead of 11

Is it a Bug ?

**Comment by tomlem** [comment]
Hi,
What exactly means -1, -1 ?
Maybe the first -1 is the value?
and maybe the second -1 stands for the sysex message number?
It's just a wild guess, but what happens as you put -1, -2 in your second sysex message, -1, -3 in the third and -1, -4 in the fourth?
What exactly means Direct, Direct in these messages?
Is there no need for checksums in these sysexes?

**Comment by Benis67** [comment]
I entered the message using the mask from the editor (see shot).
For the Sysex the parameters control number and control Value -1 -1 I don't think are used: for both the range is between -2 and +127.
Even inserting the Roland checksum (z5) the result does not change
http://www.benis.it/varie/SYSEX.PNG

**Comment by dnaldoog** [comment]
It is working for me on Windows version 5.3.201 - Which version of Ctrlr are you using?

**For Windows you are best to use 5.3.201 and macOS 5.3.198**

----
[See attached panel](https://github.com/RomanKubiak/ctrlr/files/6497577/multi.sysex._1_0_Hell-O-Kitty_2021-05-18_09-41.zip)

**Reply by Benis67** [reply]
thank you very much for the sample panel. My mistake was in putting "Midi Message Type = SysEX"; the correct value is "Midi Message Type = Multi"
No problem even with version 6.0.26
